Assets/_BForBoss/_Character/Scripts/PlayerSlideBehaviour.cs

```diff
@@ -1,8 +1,7 @@
-using ECM2.Characters;
 using ECM2.Common;
```

Comment:
Whats the purpose of removing the character using if we're still using the common namespace?

Reply:
Actually this might get at another point - what if we moved external resources into their respective modules? So ECM2 is part of the _Character module only, instead of just loose in the project. The project I'm on in my day job has a really intense package workflow, and third party assets end up in resource folder inside of their respective packages if they're needed for package functions, it could be something we could consider copying here

Reply:
so to answer ur first question. 
Because we're naming this `Character` is conflicts with the `ECM2` `Character.

Reply:
answer to the second question. 
The sole reason to keep it outside, is if we were to update the module, it'll be easier to see what needs to be updated, what doesn't.

Reply:
Yeah thats fair, I was playing around with it, and well moving ECM2 inside and replacing its asmdef with a reference file worked quite well, I did realise that we'd pretty quickly run into a dependency hell scenario with odin and dotween, since pretty much everything depends on them. We could try to bundle them with utilities since that is also a common dependency, but I think it would pretty quickly be more trouble than its worth

Reply:
odin and dotween is sort of unavoidable since you do use it everywhere.